### DISCUSSION TITLE
Folder monitoring for conversion to a defined output format

### DISCUSSION THREAD

**Discussion by xsibit666** [zelon88/HRConvert2#15] [Ideas]
Hello, is it possible to monitor a folder 
where the input format is then 
converted into a defined output format 
and stored in another folder.

**Comment by zelon88** [comment]
That's pretty much what HRConvert2 does, but on an individual basis..

Are you looking to add support for entire folders full of similar files? Similar to converting an entire folder full of .png files to .jpg? That could be interesting. 

I think this could be accomplished by having multiple sections to the "Bu;k file options" section. If images are uploaded, an image section would appear (for example). Then you could select options for converting "all uploaded images". That way you could upload 100 images at a time and change the filename. There could also be a checkbox option for "Preserve filenames" and another option for "output archive file format".

We would want to provide the output files to the user in an archive to avoid blasting them with 10's or 100's of file downloads at the same time.
